src/components/CategoryFilter.jsx

```diff
@@ -1,17 +1,24 @@
 import '../css/CategoryFilter.css';
 
-function CategoryFilter() {
+function CategoryFilter({ category, onChangeCategory }) {
+    const categoryData = ['전체', '한식', '중식', '일식', '양식', '아시안', '기타'];
```

Comment:
```suggestion
// 컴포넌트 외부에 선언
const categoryData = ['전체', '한식', '중식', '일식', '양식', '아시안', '기타'];

function CategoryFilter({ category, onChangeCategory }) {
```

`CategoryFilter` 컴포넌트가 리렌더링될 때마다 `categoryData`라는 변수가 재선언될 것 같은데 컴포넌트 외부로 빼는건 어떨까요??🙃

Reply:
함수 안에 무조건 선언을 해야한다고만 생각을 했었는데 새롭게 배워갑니다👍